src/main/resources/static/templates/layout.html

```diff
@@ -14,165 +14,332 @@
-    <script>function OptanonWrapper() { { window.dataLayer.push({ event: 'OneTrustGroupsUpdated' }); } }</script>
-    <script>
-        function setGTM(w,d,s,l,i){ w[l]=w[l]||[]; w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'}); var f=d.getElementsByTagName(s)[0],j=d.createElement(s),dl=l!='dataLayer'?'&l='+l:''; j.async=true; j.src='https://www.googletagmanager.com/gtm.js?id='+i+dl;f.parentNode.insertBefore(j,f); }
-        if (document.cookie.indexOf('OptanonConsent') > -1 && document.cookie.indexOf('groups=') > -1) { setGTM(window,document,'script','dataLayer','GTM-KMF5CLQ'); } else{ waitForOnetrustActiveGroups(); } var timer; function waitForOnetrustActiveGroups() { if (document.cookie.indexOf('OptanonConsent') > -1 && document.cookie.indexOf('groups=') > -1) { clearTimeout(timer); setGTM(window,document,'script','dataLayer','GTM-KMF5CLQ'); } else{ timer=setTimeout(waitForOnetrustActiveGroups, 250); } }
-    </script>
-    <!-- End OneTrust Cookie Consent -->
+<html
+ th:fragment="layout(page,title,description,links,content)"
+ xmlns:th="http://www.thymeleaf.org"
+>
+ <head>
+  <script
+   src="https://cdnjs.cloudflare.com/ajax/libs/jquery/3.6.1/jquery.min.js"
+   integrity="sha512-aVKKRRi/Q/YV+4mjoKBsE4x3H+BkegoM/em46NNlCqNTmUYADjBbeNefNxYV7giUp0VxICtqdrbqU7iVaeZNXA=="
+   crossorigin="anonymous"
+   referrerpolicy="no-referrer"
+  ></script>
+  <!-- OneTrust Cookie Consent -->
+  <meta
+   content="60a709d9-50a1-442d-8f59-072e08a3ddb4"
+   name="onetrust-data-domain"
+  />
+  <meta
+   content="https://tags.tiqcdn.com/utag/vmware/microsites-privacy/prod/utag.js"
+   name="microsites-utag"
+  />
+  <script src="https://www.vmware.com/files/templates/inc/utag_data.js"></script>
+  <script src="https://tags.tiqcdn.com/utag/vmware/microsites-privacy/prod/utag.sync.js"></script>
+  <script>
+   function OptanonWrapper() {
+    {
+     window.dataLayer.push({ event: 'OneTrustGroupsUpdated' });
+    }
+   }
+  </script>
+  <script>
+   function setGTM(w, d, s, l, i) {
+    w[l] = w[l] || [];
+    w[l].push({ 'gtm.start': new Date().getTime(), event: 'gtm.js' });
+    var f = d.getElementsByTagName(s)[0],
+     j = d.createElement(s),
+     dl = l != 'dataLayer' ? '&l=' + l : '';
+    j.async = true;
+    j.src = 'https://www.googletagmanager.com/gtm.js?id=' + i + dl;
+    f.parentNode.insertBefore(j, f);
+   }
+   if (
+    document.cookie.indexOf('OptanonConsent') > -1 &&
+    document.cookie.indexOf('groups=') > -1
+   ) {
+    setGTM(window, document, 'script', 'dataLayer', 'GTM-KMF5CLQ');
+   } else {
+    waitForOnetrustActiveGroups();
+   }
+   var timer;
+   function waitForOnetrustActiveGroups() {
+    if (
+     document.cookie.indexOf('OptanonConsent') > -1 &&
+     document.cookie.indexOf('groups=') > -1
+    ) {
+     clearTimeout(timer);
+     setGTM(window, document, 'script', 'dataLayer', 'GTM-KMF5CLQ');
+    } else {
+     timer = setTimeout(waitForOnetrustActiveGroups, 250);
+    }
+   }
```

Comment:
I'm only uncertain if it's ok for us to leave this non-minimised. If the build minimises the scripts then great...